ups/sims_seeingModel.table

```diff
@@ -0,0 +1,14 @@
+setupRequired(utils >= 4.6.0.0)
+# These packages are optional for running sims_seeingModel, but required for tests.
+# Note that 'eups distrib' installs and sets up setupOptional packages.
+# TimeHandler lives in sims_utils
+setupOptional(sims_utils)
```

Comment:
Ah -- details. It's optional for *running* the package, but it's required for unit tests (because of the fact that timeHandler is now in sims_utils). 
Happily (?) when you run "eups distrib install", setupOptional packages are handled as setupRequired -- so sims_utils will be setup when running the unit tests in a normal install.
My thought on this is that if you're running the unit tests by hand (via scons or pytest), then you should be able to setup sims_utils yourself anyway.

So: yes, it's optional although required for unit tests.